src/app/(landing-page)/blog/[posts]/page.tsx

```diff
@@ -24,21 +25,23 @@ const Post = ({ params }: { params: { posts: string } }) => {
       if (docSnap.exists()) {
         setLoadedDoc(docSnap.data() as IBlog);
       } else {
-        notFound();
+        router.push(`/${endpoints.paths.error[404]}`);
```

Comment:
endpoints.paths.error.404 should already have a leading / so you don't need to format the string